src/react-components/ui-root.js

```diff
@@ -54,6 +54,8 @@ async function grantedMicLabels() {
 
 const AUTO_EXIT_TIMER_SECONDS = 10;
 
+const isOculusBrowser = navigator.userAgent.match(/Oculus/);
```

Comment:
this should probably use the `isInHMD` check from vr device caps, which will be true for oculus browser and fxr

Reply:
Hmm, will `isInHMD` not also be true for Vive/Rift, and if not, should we rename it to something less confusing?